src/util/aggregate-flags.ts

```diff
@@ -0,0 +1,18 @@
+import {FlagInput, FlagOutput} from '../interfaces/parser'
+import {boolean} from '../flags'
+
+export const json = boolean({
```

Comment:
what happens if this is not exported?

Reply:
It doesn't need to be exported. I'll change it